### ISSUE TITLE
Anybody for a custom UNICODE encoding?

### ISSUE THREAD

**Issue by misyntropy** [zedapp/zed#486]
is anyone interested in allowing custom encodings for Unicode itself? By that I mean, an alternative unicode encoding to UTF-8, UTF-16...

**Comment by zefhemel** [comment]
Yeah, I'd be interested in that, no idea what it'd entail to do it, though.

**Comment by milindsmart** [comment]
It shouldn't entail any large changes if a text file is loaded into memory as UTF-32. The idea is to develop modules that support a new text-encoding and simply drop them into an appropriate folder. That encoding would become available to the user alongside UTF-16BE or LE or UTF-8 with BOM (say).
The module itself would be a (set of) functions to map a set of bytes to a unicode codepoint.

**Comment by zefhemel** [comment]
Is this not a duplicate of #468?

**Comment by milindsmart** [comment]
Technically yes. May not be in practice, because i'm suggesting enabling easy addition of custom encodings, as opposed to just widely used ones.
